src/main/java/seedu/trackermon/logic/commands/CommandResult.java

```diff
@@ -17,21 +17,33 @@
     /** The application should exit. */
     private final boolean exit;
 
+    /** The list will be modified **/
+    private final boolean modifyList;
```

Comment:
Maybe can make it sound more boolean-like: isModifiedList? JustCurious how come we don't need this boolean before but not we need?

Reply:
Noted!